__tests__/app.test.js

```diff
@@ -0,0 +1,21 @@
+const request = require("supertest");
+const app = require("../app");
+const data = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const db = require("../db//connection");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('GET api/topics', () => {
+    test('Status-200: responds with an array containing all topics', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({ body }) => {
+                console.log({body});
+                expect(body.topics).toHaveLength(3)
+                expect(Array.isArray(body.topics)).toBe(true)
+            })
+    })
```

Comment:
We can also add a sad path test here. What if someone tries to access the wrong endpoint e.g. api/banana.